include/handler.h

```diff
@@ -617,10 +614,10 @@ void handler::parallel_for_kernel_and_reductions(range<Dims> global_range, id<Di
 		} else if constexpr(!CELERITY_FEATURE_SCALAR_REDUCTIONS && sizeof...(reductions) > 1) {
 			static_assert(detail::constexpr_false<Kernel>, "DPC++ currently does not support more than one reduction variable per kernel");
 		} else if constexpr(std::is_same_v<KernelFlavor, detail::simple_kernel_flavor>) {
-			detail::invoke_sycl_parallel_for<KernelName>(cgh, chunk_range, detail::make_sycl_reduction(cgh, reductions)...,
-			    detail::bind_simple_kernel(kernel, global_range, global_offset, chunk_offset));
+			detail::invoke_sycl_parallel_for<KernelName>(
```

Comment:
Reminder for the future that we should get rid of this path entirely as it's fundamentally broken.

Reply:
(This is now being addressed by #163)